medical/data/ir_sequence_data.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```

Comment:
Shouldn't we follow v10 standard?

`<odoo noupdate="1">` 

And remove the `<data>`